modules/ocf_ns/templates/named.conf.options.erb

```diff
@@ -39,11 +39,7 @@ options {
 
   dnssec-enable yes;
   dnssec-lookaside auto;
-
-  // Don't validate DNSSEC yet, since mesos-dns doesn't support it
-  // dnssec-must-be-secure doesn't work either unfortunately :(
-  // Neither does using disable-algorithms on only the mesos zone
-  dnssec-validation no;
+  dnssec-validation yes;
```

Comment:
I'm a bit concerned about including this change in with the rest of these changes, having it in a separate commit would be better overall in case we need to revert just this part. Anyway, not a big deal since we can just change this part of this file to revert if needed.

Reply:
I tested this on pestilence for about 15 minutes, and everything seemed to work. Are there still issues with dnssec (https://github.com/ocf/dns/issues/25)?